chart/traefik/templates/ingress.yaml

```diff
@@ -0,0 +1,38 @@
+{{- if .Values.traefik.ingress.alb.enabled }}
```

Comment:
@shaharblanksela 
@germanilia 
@ItzhakLevi 

Hey Shachar I know we talked and you told me that you are using the existing traefik ingress controller we created. please notice that this is an ALB and we discussed on changing it to ELB.

Reply:
What about local development ingress??? If I'm not part of Amplication and I want to run the charts? or part of Amplication but just want to run locally